app/models/early_career_payments/eligibility.rb

```diff
@@ -73,7 +73,9 @@ def policy
     def qualification_name
       return qualification.gsub("_itt", " ITT") if qualification.split("_").last == "itt"
 
-      qualification.humanize.downcase
+      qualification_attained = qualification.humanize.downcase
+
+      qualification_attained == "assessment only" ? qualification_attained : qualification_attained + " qualification"
```

Comment:
Seems odd to add ` qualification` to the qualification name on the  business side of things, would be like `Mark Sutton name`. Sounds like something that should be handled on the view?